frontend/src/metabase/parameters/components/WidgetStatusIcon.tsx

```diff
@@ -8,20 +7,16 @@ type Props = {
 };
 
 export function WidgetStatusIcon({ name, onClick }: Props) {
-  const classes = cx(
-    "flex-align-right flex-no-shrink",
-    name === "close" && "cursor-pointer",
-  );
+  const classes = cx("flex-align-right flex-no-shrink", {
+    "cursor-pointer": name === "close",
+  });
 
-  const handleOnClick = useCallback(
-    e => {
-      if (onClick) {
-        e.stopPropagation();
-        onClick();
-      }
-    },
-    [onClick],
-  );
+  const handleOnClick = (e: React.MouseEvent) => {
```

Comment:
1. We always destruct the React imports
2. `event` > `e`

```suggestion
  const handleOnClick = (event: MouseEvent) => {
```